ptmrio.phrasevault version 2.2.2

ptmrio opened the pull request:
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Is there a linked Issue? [https://github.com/microsoft/winget-pkgs/issues/323262](https://github.com/microsoft/winget-pkgs/issues/323262)

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.10 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.10.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/319203)

ptmrio committed:
ptmrio.phrasevault version 2.2.2

ptmrio committed:
Update ptmrio.phrasevault.installer.yaml

ptmrio committed:
Add silent installation switches to installer YAML

Added installer switches for silent installation.

ptmrio committed:
Format YAML file for ptmrio.phrasevault installer

ptmrio committed:
Update InstallerSha256 in ptmrio.phrasevault installer

Updated InstallerSha256 for PhraseVault installer.

stephengillie commented:
Automatic Validation ended with: 
> 
> Executable C:\Users\Validator\AppData\Local\PhraseVault\Update.exe returned exit code: -1
> Executable C:\Users\Validator\AppData\Local\PhraseVault\PhraseVault.exe returned exit code: -1
> Executable C:\Users\Validator\AppData\Local\PhraseVault\app-2.2.2\squirrel.exe returned exit code: -1 
 
 | Hex | Dec | Inverted Dec | Symbol | Description | 
 | --- | --- | --- | --- | --- | 
  | FFFFFFFF | -1 | 4294967295 | Cancel |  | 


(Automated response - build 1167.)

ptmrio committed:
Update ptmrio.phrasevault installer YAML

Added comment regarding SmartScreen error.

ptmrio committed:
Revise comments in ptmrio.phrasevault.installer.yaml

Updated comments to reflect the status of Microsoft Smart Screen submission.

ptmrio committed:
Update ptmrio.phrasevault installer YAML file

stephengillie commented:
Automatic Validation ended with: 
> 
> Executable C:\Users\Validator\AppData\Local\PhraseVault\Update.exe returned exit code: -1
> Executable C:\Users\Validator\AppData\Local\PhraseVault\PhraseVault.exe returned exit code: -1
> Executable C:\Users\Validator\AppData\Local\PhraseVault\app-2.2.2\squirrel.exe returned exit code: -1 
 
 | Hex | Dec | Inverted Dec | Symbol | Description | 
 | --- | --- | --- | --- | --- | 
  | FFFFFFFF | -1 | 4294967295 | Cancel |  | 


(Automated response - build 1178.)

ptmrio committed:
Remove redundant comments in installer YAML

ptmrio committed:
Add submission details to PhraseVault installer manifest

Updated the installer manifest for PhraseVault to include submission ID and status information.

stephengillie commented:
Automatic Validation ended with: 
> 
> Executable C:\Users\Validator\AppData\Local\PhraseVault\Update.exe returned exit code: -1
> Executable C:\Users\Validator\AppData\Local\PhraseVault\PhraseVault.exe returned exit code: -1
> Executable C:\Users\Validator\AppData\Local\PhraseVault\app-2.2.2\squirrel.exe returned exit code: -1 
 
 | Hex | Dec | Inverted Dec | Symbol | Description | 
 | --- | --- | --- | --- | --- | 
  | FFFFFFFF | -1 | 4294967295 | Cancel |  | 


(Automated response - build 1178.)

ptmrio committed:
Update ptmrio.phrasevault.installer.yaml

ptmrio committed:
Update installer.yaml with upgrade behavior

Added UpgradeBehavior and InstallerSuccessCodes fields.
